src/GeositeFramework/js/Plugin.js

```diff
@@ -235,8 +235,10 @@ require(['use!Geosite',
                     showValueKey = pluginObject.toolbarName + " showinfographic";
                 if (typeof localStorage[showValueKey] !== 'undefined') {
                     return localStorage[showValueKey] === 'true';
-                }
-                return true;
+                } else {
+					localStorage.setItem(showValueKey,pluginObject.showInfographicOnStart);
+					return localStorage[showValueKey] === 'true';
+				}
```

Comment:
Ok sounds good.  You mean:

return localStorage[showValueKey] === 'true';

Can be replaced with:

return pluginObject.showInfographicOnStart;

We still need set the localStorage variable though.

Do I make the change?

Thanks